widget/src/svg.rs

```diff
@@ -166,7 +167,11 @@ where
                 ..bounds
             };
 
-            let appearance = theme.appearance(&self.style);
+            let appearance = if is_mouse_over {
+                theme.appearance(&self.style)
+            } else {
+                theme.hovered(&self.style)
+            };
```

Comment:
These branches need to be switched.

Reply:
i already switched. there was ask me to separate 2 changes.

Reply:
I mean `hovered` should be called if `is_mouse_over`, not in the `else` branch.

Reply:
```suggestion
            let appearance = if is_mouse_over {
                theme.hovered(&self.style)
            } else {
                theme.appearance(&self.style)
            };
```